scripts/gui/microbe_editor.mjs

```diff
@@ -921,15 +935,16 @@ function moveToPatchClicked(){
     if(!patchMoveAllowed(selectedPatch.id))
```

Comment:
`patchMoveAllowed` needs a check for freebuild, otherwise it will return false and prevent the move.
In `patchMoveAllowed`:
```
    if(currentPatchId == targetId){
        return false;
    }

    // Freebuild, allow movement to anywhere
    if(!limitMovesPerSession)
        return true;
```

Reply:
If `limitMovesPerSession` is set to false then in `patchMoveAllowed()` 
this statment is never invoked:
`if(limitMovesPerSession)
        alreadyMovedThisSession = currentPatchId != patchIdOnEnter;`

as consequence  `alreadyMovedThisSession ` keep stay to default value `false` and so we can keep moving.

I'm gonna test it again, maybe i miss something and you're right.

Reply:
My undestanding is that @Dak2896 is right. At least that's how I wanted to write the logic and make it easy to disable the restriction on number of moves.

Reply:
I made my own version (before realizing it had already been done). It didn't work until I added that check. Just trying to help.